src/Core/Components/NavMenu/FluentNavGroup.razor

```diff
@@ -0,0 +1,69 @@
+﻿@namespace Microsoft.Fast.Components.FluentUI
+@inherits FluentNavBase
+@using Microsoft.AspNetCore.Components.Rendering
+@using Microsoft.AspNetCore.Components.Routing
+
+
+@if (NavMenuExpanded || HasIcon)
+{
+    <div @attributes="AdditionalAttributes" class="@ClassValue" disabled="@Disabled" style="@StyleValue">
+        <div class="positioning-region">
+            <div class="content-region">
+                @if (!string.IsNullOrEmpty(Href))
+                {
+                    <NavLink class="fluent-nav-link"
+                             @attributes="@Attributes"
+                             Match="@Match"
+                             ActiveClass="@ActiveClass">
+                        @_renderContent
+                    </NavLink>
+                    @_renderButton
+                }
+                else
+                {
+                    <div class="fluent-nav-link notactive" tabindex="0" @onclick="ToggleExpandedAsync" @onkeydown="@HandleExpanderKeyDownAsync">
+                        @_renderContent
+                        @_renderButton
+                    </div>
+                }
+            </div>
+        </div>
+        <FluentCollapsibleRegion @bind-Expanded="@Expanded" MaxHeight="@MaxHeight" Class="items" role="group">
+            <FluentNavMenu>
+                @ChildContent
+            </FluentNavMenu>
+        </FluentCollapsibleRegion>
+    </div>
+}
+@code {
+    private void RenderContent(RenderTreeBuilder __builder)
+    {
+        @if (Icon is not null)
+        {
+            <FluentIcon Value="@Icon" Width="20px" Color="@IconColor" Class="fluent-nav-icon" />
```

Comment:
Do we need to set `20px` in a C# constant or in a CSS variable to allow future updates (of dev changes)?

Reply:
Added `internal const string ICON_WIDTH = 20px;`